tyk-single-dc/README.md

```diff
@@ -0,0 +1,208 @@
+## Tyk Tyk single dc
+
+`tyk-single-dc` provides the default deployment of a Tyk Single Data Center. It will deploy the data center components with the settings provided in the values.yaml file.
```

Comment:
```suggestion
`tyk-single-dc` provides the default deployment of a Tyk Pro on single data center. It will deploy all required Tyk stack components with the settings provided in the values.yaml file.
```